packages/fast-sketch-library/src/index.ts

```diff
@@ -0,0 +1,162 @@
+import * as puppeteer from "puppeteer";
+import { Browser, Page } from "puppeteer";
+import * as fs from "fs";
+import * as path from "path";
+import { Page as SketchPage } from "@brainly/html-sketchapp";
+
+/**
+ * Store in-page script content as a string to be loaded into the browser
+ */
+const aSketchPage: string = fs.readFileSync(path.resolve(__dirname, "./aSketchPage.js")).toString();
+
```

Comment:
Should interface comments be added to fully understand the purpose of the interface itself?

Reply:
good call  - updated.